lib/ufe/Global.cpp

```diff
@@ -22,7 +22,6 @@
 #include "UsdTransform3dHandler.h"
 #include "UsdSceneItemOpsHandler.h"
 
-#include <ufe/rtid.h>
```

Comment:
Looks like we're actually adding a usage of `Ufe::Rtid` at line 130, so I'm not sure why we're taking this include out? I know it was added to the header, but since there's usage in both files, I'd prefer to see the include in both files.

Reply:
Not a huge deal, to be sure, but isn't that violating Don't Repeat Yourself?  If maintenance removes the use of Ufe::Rtid, I'd now have two places to remove the include of ufe/rtid.h.